tests/library/test_recipe.py

```diff
@@ -176,7 +176,7 @@ def test_standard_recipe_production_with_demos(self):
             ' "choices": ["yes", "not", "maybe"],'
             ' "answer": "maybe",'
             ' "options": [" A", " B", " C"],'
-            ' "metadata": {"template": "templates.qa.multiple_choice.with_topic.lm_eval_harness"}'
```

Comment:
Notice the data_classificaiton policy now appears in two places.  In the metadata and in a dedicated field (see line 183).